src/handlers/scheduledEventHandler.ts

```diff
@@ -0,0 +1,53 @@
+import { env } from "../typeDefinitions/default.types";
+import { taskOverDueDiscordMembers } from "../utils/taskOverDueDiscordMembers";
+import * as error from "../constants/responses";
+import { getDiscordIds } from "../utils/getDiscordIds";
+import { RDS_TRACKING_CHANNEL_URL } from "../constants/urls";
+
+export async function send(env: env): Promise<void> {
+  try {
+    const assigneeIds: string[] | string = await taskOverDueDiscordMembers();
+
+    //A user might have more than one task which are running red
+    //so to mention them just once, we are using Set to filter out
+    const discordIds: string[] | string = await getDiscordIds(assigneeIds);
+    const uniqueDiscordIds = [...new Set(discordIds)];
+
+    //notifying the two users with the authority.
+    let str = `<@154585730465660929> <@1040700289348542566>\nThese people have their task running red:\n`;
+
+    let forFormatting = 0;
+    for (let i = 0; i < uniqueDiscordIds?.length; i++) {
+      const s = `<@${uniqueDiscordIds[i]}> `;
+      str += s;
+      forFormatting++;
+      if (forFormatting === 3) {
```

Comment:
Don't you have to re-assign `forFomatting` to 0?

Reply:
This is to add a new line after 3 users/ line, is it not?

Reply:
good catch! Yes this is for having 3 users/line